testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/grpc/GrpcToJaxrsTest.java

```diff
@@ -0,0 +1,805 @@
+import jaxrs.example.CC1_proto.ServletInfo;
+import jaxrs.example.CC1_proto.gCookie;
+import jaxrs.example.CC1_proto.gHeader;
+import jaxrs.example.CC1_proto.gString;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC2;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC3;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC4;
+import jaxrs.example.CC1_proto.org_jboss_resteasy_example___CC5;
+
+/**
+ * @tpSubChapter Jaxrs implementation
+ * @tpChapter Integration tests
+ * @tpTestCaseDetails RESTEASY-1531
+ * @tpSince RESTEasy 3.1.0
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+public class GrpcToJaxrsTest
+{
+   protected static final Logger log = Logger.getLogger(GrpcToJaxrsTest.class.getName());
+
+   @Deployment
+   public static Archive<?> deploy()
+   {
+      WebArchive war = TestUtil.prepareArchive(GrpcToJaxrsTest.class.getSimpleName());
+      war.addClass(io.grpc.netty.shaded.io.netty.channel.group.ChannelMatchers.class);
+      war.addClass(com.google.common.util.concurrent.internal.InternalFutureFailureAccess.class);
+      war.setManifest(new StringAsset("Manifest-Version: 1.0\n"
+//            + "Dependencies: com.google.guava services,org.jboss.resteasy.resteasy-grpc-provider services\n"));
+      + "Dependencies: com.google.guava services\n"));
+      war.merge(ShrinkWrap.createFromZipFile( WebArchive.class, TestUtil.resolveDependency("jaxrs.example:jaxrs.example.grpc:war:0.0.2-SNAPSHOT")));
+      TestUtil.addOtherLibrary(war, "org.jboss.resteasy:grpc-bridge:jar:6.1.0-SNAPSHOT");
+      TestUtil.addOtherLibrary(war, "com.github.javaparser:javaparser-symbol-solver-core:jar:3.24.2");
+      TestUtil.addOtherLibrary(war, "com.github.javaparser:javaparser-core:jar:3.24.2");
+      TestUtil.addOtherLibrary(war, "org.wildfly:wildfly-weld-jakarta:jar:26.1.0.Final");
+      TestUtil.addOtherLibrary(war, "org.wildfly:wildfly-weld-common-jakarta:jar:26.1.0.Final");
+      TestUtil.addOtherLibrary(war, "org.wildfly.core:wildfly-server:jar:18.1.0.Final");
+      TestUtil.addOtherLibrary(war, "com.google.protobuf:protobuf-java:jar:3.17.3");
+      TestUtil.addOtherLibrary(war, "io.grpc:grpc-api:1.39.0");
+      TestUtil.addOtherLibrary(war, "io.grpc:grpc-context:1.39.0");
+      TestUtil.addOtherLibrary(war, "io.grpc:grpc-core:1.39.0");
+      TestUtil.addOtherLibrary(war, "io.grpc:grpc-netty-shaded:1.39.0");
+      TestUtil.addOtherLibrary(war, "io.grpc:grpc-protobuf:1.39.0");
+      TestUtil.addOtherLibrary(war, "io.grpc:grpc-protobuf-lite:1.39.0");
+      TestUtil.addOtherLibrary(war, "io.grpc:grpc-stub:1.39.0");
+      TestUtil.addOtherLibrary(war, "io.perfmark:perfmark-api:0.23.0");
+      TestUtil.addOtherLibrary(war, "org.jboss.weld.se:weld-se-core:jar:5.0.0.CR2");
+      TestUtil.addOtherLibrary(war, "org.jboss.weld.environment:weld-environment-common:jar:5.0.0.CR2");
+      TestUtil.addOtherLibrary(war, "com.google.guava:failureaccess:jar:1.0.1");
```

Comment:
In general none of these versions should be hard-coded. It makes it very difficult to upgrade the component.

Reply:
@jamezp, the latter three of these are gone. Right now, GrpcToJaxrsTest fails without perfmark-api. More generally, I need to get the right jars either in

 * the grpc-bridge-runtime module in WildFly, or
 * the generated WAR lib directory

I've been having some trouble with it and might need to consult with you.